src/components/shared/sidebar/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import { useMutation } from "react-query";
+import { HomeIcon, InboxIcon, PlusCircleIcon } from "@heroicons/react/24/outline";
+import Router from "next/router";
+
+import { logout } from "@/pages/api";
+import { useStore } from "@/pages/store/store";
+import NavItem from "./NavItem";
+
+export default function Sidebar() {
+  const { token, user } = useStore();
+  const [selected, setSelected] = useState("Menu");
+
+  const mutationLogout = useMutation(() => logout(token), {
+    onSuccess: () => {
+      useStore.getState().setToken("");
+      useStore.getState().setUser(undefined);
+      Router.push("/auth");
+    },
+  });
+
+  return (
+    <>
```

Comment:
why the fragment? you only need this when trying to export two children without a parent but there you're only exporting a single div